packages/convert-imports.js

```diff
@@ -0,0 +1,25 @@
+const glob = require('glob')
+const fs = require('fs')
+
+var getDirectories = function (callback) {
+  glob('./common/src/**/*', callback)
+}
+
+getDirectories(async function (err, res) {
```

Comment:
Same here, a JSDoc would help. Also a bit weird to have the same name as the above?